employees.html

```diff
@@ -0,0 +1,513 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/font.css" />
+    <link rel="stylesheet" href="css/common.css" />
+    <link rel="stylesheet" href="css/Employees.css" />
+
+    <title>Employees-dashboard</title>
+  </head>
+  <body>
+    <div class="container">
+      <section class="sidebar">
+        <div class="dashboard">
+          <figure class="logo">
+            <span class="icon-TezoLogo"
+              ><span class="path1"></span><span class="path2"></span
+              ><span class="path3"></span><span class="path4"></span
+              ><span class="path5"></span
+            ></span>
+            <img src="images/handle.png" alt="handle" class="sidebar-handle" />
+          </figure>
+          <div class="all-dashboard">
+            <div class="all-dashboard-heading">
+              <div class="all-dashboard-heading-line"></div>
+              <p>ALL</p>
+            </div>
+            <ul>
+              <li>
+                <img src="images\horizontal-nav\Dashboard.svg" alt="" />
+                Dashboard
+              </li>
+              <div class="all-dashboard-selected-menu">
+                <li>
+                  <img src="images\horizontal-nav\Employees.svg" alt="" />
+                  Employees
+                </li>
+                <div class="Dashboard-selection-highlight"></div>
+              </div>
+            </ul>
+          </div>
+          <div class="role-management-dashboard">
+            <div class="role-management-dashboard-heading">
+              <div class="role-management-dashboard-heading-line"></div>
+              <p>ROLE / USER MANAGEMENT</p>
+            </div>
+            <ul>
+              <div class="roles-selection-menu">

```

Comment:
```suggestion
              <li class="roles-selection-menu">

```

Reply:
div is needed to display the content as per the UI.